Xamarin.Forms.Platform.Android/Renderers/ShellItemRenderer.cs

```diff
@@ -108,11 +110,10 @@ protected virtual bool ChangeSection(ShellSection shellSection)
 
 		protected virtual Drawable CreateItemBackgroundDrawable()
 		{
-			var stateList = ColorStateList.ValueOf(Color.Black.MultiplyAlpha(0.2).ToAndroid());
-			return new RippleDrawable(stateList, new ColorDrawable(AColor.White), null);
+			return BottomNavigationViewUtils.CreateItemBackgroundDrawable();
 		}
 
-		protected virtual BottomSheetDialog CreateMoreBottomSheet(Action<ShellSection, BottomSheetDialog> selectCallback)
+		protected virtual BottomSheetDialog CreateMoreBottomSheet(Action<int, BottomSheetDialog> selectCallback)
```

Comment:
api break, but probably not affecting anyone

Reply:
added  back as obsolete